elementary/monitor/data_monitoring.py

```diff
@@ -100,7 +101,15 @@ def _get_user_id(email: str) -> str:
             user_id = self.slack_client.get_user_id_from_email(email)
             return f"<@{user_id}>" if user_id else email
 
-        if isinstance(emails, list) and emails != []:
+        def _parse_str_to_list(emails: str) -> List[str]:
+            try:
+                return ast.literal_eval(emails)
+            except:
+                return list(emails.split(","))
+
+        if isinstance(emails, str):
+            emails = _parse_str_to_list(emails)
```

Comment:
I think that we can simplifier the code a bit in the following way:
Instead of doing the if statement like that, you can do `emails = _parse_str_to_list(emails) if isinstance(emails, str) else emails`.
That way emails will always be a list and you could skip the next if statement and always run:
```python
ids = [
    _get_user_id(email) if _regex_match_owner_email(email) else email
    for email in emails
]
parsed_ids_str = prettify_json_str_set(ids)
return parsed_ids_str
```

Reply:
The finale code will look like:
```python
def _parse_str_to_list(emails: str) -> List[str]:
    try:
        return ast.literal_eval(emails)
    except:
        return list(emails.split(","))

emails = _parse_str_to_list(emails) if isinstance(emails, str) else emails
ids = [
    _get_user_id(email) if _regex_match_owner_email(email) else email
    for email in emails
]
parsed_ids_str = prettify_json_str_set(ids)
return parsed_ids_str
```